src/components/navbar/Navbar.jsx

```diff
@@ -7,32 +7,24 @@ import { FiSearch } from "react-icons/fi";
 import "./Navbar.css";
 import {
   Link,
+  NavLink,
   useLocation,
-  useNavigate,
   useSearchParams,
 } from "react-router-dom";
 
 import { useAuth } from "../../context/auth/AuthContext";
 
-const Navbar = ({ hideHamburgerMenu , toogle}) => {
+const Navbar = ({ hideHamburgerMenu, toogle }) => {
   const { authState } = useAuth();
-
-  const handleClick = () => toogle((v) => !v);
-  const navigate = useNavigate();
   const { pathname } = useLocation();
+  const handleClick = () => toogle((v) => !v);
```

Comment:
have clear naming

Reply:
It is a Boolean value so, I will change it to `isTrue`